share/models/people.py

```diff
@@ -16,25 +16,17 @@ def __str__(self):
         return self.email
 
 
-class Identifier(ShareObject):
```

Comment:
I've gone back and forth on this, and went with a single table mostly because it was easier to name (`Link` sounds more like a relation than an identifier, and `PersonIdentifier`/`WorkIdentifier` seemed cumbersome...). Separate identifier tables will almost certainly have better performance, I think, so I'll split them up.